app/models/journal.rb

```diff
@@ -132,7 +132,13 @@ def save_data
 
   def touch_journable
     if journable && !journable.changed?
-      journable.touch
+      # Not using touch here on purpose,
+      # as to avoid changing lock versions on the journables for this change
+      time = journable.send(:current_time_from_proper_timezone)
+      attributes = journable.send(:timestamp_attributes_for_update_in_model)
+
+      timestamps = Hash[attributes.map{|column| [column, time]}]
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.